src/Database/DBStructure.php

```diff
@@ -1818,6 +1818,20 @@ public static function definition() {
 						"PRIMARY" => ["uid", "iid"],
 						]
 				];
+		$database["verify"] = [
```

Comment:
Table naming: is this used by other protocols than OpenWebAuth? If no, it probably should refer to it.